crates/ere-zisk/src/lib.rs

```diff
@@ -284,13 +295,17 @@ impl zkVM for EreZisk {
     }
 }
 
+/// Serialize `Input` into sequence of bytes.
+///
+/// Because ZisK doesn't provide stdin API so we need to handle multiple inputs,
+/// the current approach naively serializes each `InputItem` individually, then
+/// concat them into single `Vec<u8>`.
 fn serialize_inputs(inputs: &Input) -> Result<Vec<u8>, bincode::Error> {
     inputs.iter().try_fold(Vec::new(), |mut acc, item| {
         match item {
-            InputItem::Object(obj) => {
-                bincode::serialize_into(&mut acc, &**obj)?;
-            }
-            InputItem::SerializedObject(bytes) | InputItem::Bytes(bytes) => acc.extend(bytes),
+            InputItem::Object(obj) => bincode::serialize_into(&mut acc, &**obj)?,
+            InputItem::SerializedObject(bytes) => acc.extend(bytes),
+            InputItem::Bytes(bytes) => bincode::serialize_into(&mut acc, bytes)?,
```

Comment:
We need to treat the bytes as object and serialize it as well, otherwise the deserializer doesn't know the length of the bytes.

`zisk`'s guest program in `zkevm-benchmark-workload` also needs to be updated.